Makefile-tests.am

```diff
@@ -287,6 +294,17 @@ tests/%-symlink-stamp: % Makefile
 	ln -sf "$${real_bin}" tests/$*; \
 	touch $@
 
+# See above comment on binding the tests to be either installed or not.
+if ENABLE_INSTALLED_TESTS_EXCLUSIVE
+installed_test_scripts = $(_installed_or_uninstalled_test_scripts)
```

Comment:
I think we can probably assume no one is going to try to dist with --enable-installed-tests=exclusive, but this should probably be `dist_installed_test_scripts`.